tests/conftest.py

```diff
@@ -158,6 +158,28 @@ def multidict_getversion_callable(multidict_module: ModuleType) -> Callable:
     return multidict_module.getversion
 
 
+def write(cls: Any, proto: int) -> bytes:
+    d = cls([("a", 1), ("a", 2)])
+    return pickle.dumps(d, proto)
+
+
+@pytest.fixture
+def in_memory_pickle_classes(multidict_module: ModuleType) -> dict[str, bytes]:
+    """Generates a dict for in-memory storage of pickled classes"""
```

Comment:
A docstring title must use an imperative mood and be a full sentence with a trailing period.

Reply:
Fixed